tasks/kmt.py

```diff
@@ -127,37 +127,35 @@ def gen_config_from_ci_pipeline(
     """
     Generate a vmconfig.json file with the VMs that failed jobs in the given pipeline.
     """
-    gitlab = Gitlab("DataDog/datadog-agent", str(get_gitlab_token()))
+    repo = get_gitlab_repo()
```

Comment:
nitpick (just to illustrate it's this one you get by default)
```suggestion
    datadog_agent = get_gitlab_repo()
```